cpu/atmega_common/periph/gpio.c

```diff
@@ -148,6 +148,10 @@ int gpio_init_int(gpio_t pin, gpio_mode_t mode, gpio_flank_t flank,
     /* clear global interrupt flag */
     cli();
 
+#ifdef CPU_ATMEGA328P
+    pin_num -= 2;
```

Comment:
I don't question this, but could you please give a short reference/reasoning for this? Thx 😄

Reply:
Interrupt pins in atmega25.../12... are starting at 0 (e.g. PD0) but in atmega328 start at 2 (e.g. PC2). I'll add a comment :-) :+1: